tests/aws/services/apigateway/apigateway_fixtures.py

```diff
@@ -198,11 +198,17 @@ class UrlType(Enum):
 
 
 def api_invoke_url(
```

Comment:
at some point, we should think about migrating this as a real fixture, so we could use the `region_name` fixture as well. Could be useful! it's used everywhere 😬